platform/windows/display_server_windows.cpp

```diff
@@ -1093,6 +1093,7 @@ bool DisplayServerWindows::can_any_window_draw() const {
 	_THREAD_SAFE_METHOD_
 
 	for (Map<WindowID, WindowData>::Element *E = windows.front(); E; E = E->next()) {
+		if (E->key() == -1) continue;
```

Comment:
Should be `INVALID_WINDOW_ID` (which is `-1`) instead of `-1`.

Reply:
Some missing checks I found:

https://github.com/godotengine/godot/blob/d2d359d67c5c35351ed953448430536c57db1522/platform/windows/display_server_windows.cpp#L498-L500

`_create_window` can return -1

https://github.com/godotengine/godot/blob/d2d359d67c5c35351ed953448430536c57db1522/platform/windows/display_server_windows.cpp#L1788-L1795

Can potentially use -1 if window is not in the map (window is created but not added to the map, or removed from the map but still closing, not sure if any of this is possible).